infrastructure/deploy.tf

```diff
@@ -0,0 +1,96 @@
+resource "azurerm_resource_group" "example" {
+  name     = "zenml-test"
```

Comment:
Feels like maybe this might clash at some point. Maybe call it `zenml-github-test` or something that is unlikely for someone else to create?